avocado/utils/lv_utils.py

```diff
@@ -54,6 +55,27 @@ def get_diskspace(disk):
     raise LVException('Error in finding disk space')
 
 
+def get_disks_total_space(disks):
+    """
+    Get the entire disk space of a given disk(s).
+    :param str disk: name of the disk(s) to find the free space of
+    :returns: size in bytes
+    :rtype: int
+    :raises: :py:class:`LVException` on failure to find disk space
+    """
+    size = 0
+    for disk in disks.split(" "):
+        result = process.run('fdisk -l %s' % disk,
+                             env={"LANG": "C"}, sudo=True).stdout_text
+        results = result.splitlines()
+        for line in results:
+            if line.startswith('Disk ' + disk):
+                size = size + int(re.findall(r", (.*?) bytes", line)[0])
```

Comment:
Please raise an exception here, or at least store the disk(s) for which size is not got and mention those in exception message